docs/yum.8

```diff
@@ -858,12 +858,12 @@ the installroot (and thus. will be empty, before creation).
 .br
 Configuration Option: \fBinstallroot\fP
 .IP "\fB\-\-enablerepo=repoidglob\fP"
-Enables specific repositories by id or glob that have been disabled in the 
+Temporarily enables specific repositories by id or glob that have been disabled in the 
 configuration file using the enabled=0 option.
 .br
 Configuration Option: \fBenabled\fP
 .IP "\fB\-\-disablerepo=repoidglob\fP"
-Disables specific repositories by id or glob. 
+Temporarily disables specific repositories by id or glob. 
```

Comment:
Same as above.

Reply:
OK, I presume I do not need to push a change, you can just not use it.

Reply:
Yeah, I just noticed they are different commits. I'll do the honors then, thanks.